src/community/__stories__/data-card-advanced-story.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import ResponsiveLayout from '../../responsive-layout';
+import Box from '../../box';
+import {ButtonLink, ButtonPrimary} from '../../button';
+import Tag from '../../tag';
+import DataCardAdvanced from '../../data-card-advanced';
+import imgExample from '../../__stories__/images/avatar.jpg';
+import Image from '../../image';
+import StackingGroup from '../../stacking-group';
+
+import type {TagType} from '../../tag';
+
+export default {
+    title: 'Community/DataCardAdvanced',
+};
+
+type Args = {
+    headline: string;
+    headlineType: TagType;
+    pretitle: string;
+    title: string;
+    subtitle: string;
+    description: string;
+    footerText: string;
+    showStackingGroup: boolean;
+    showFooterImage: boolean;
+    showHeadline: boolean;
+    showButtonPrimary: boolean;
+    showButtonLink: boolean;
```

Comment:
Please, follow similar solutions already implemented in other component stories
![image](https://github.com/Telefonica/mistica-web/assets/6722153/bc7e4aac-ac02-4840-9ccc-9483790d65c4)

[Data Card Story](https://mistica-kbglt83lx-tuentisre.vercel.app/?path=/story/components-cards-data-card--default)

This way, it is easier to understand the possible actions that the card can take